components/brave_rewards/resources/ui/components/donate/index.tsx

```diff
@@ -94,6 +94,31 @@ export default class Donate extends React.PureComponent<Props, State> {
     }
   }
 
+  generateMissingFunds = () => {
+    const {
+      addFundsLink,
+      onlyAnonWallet
+    } = this.props
+
+    let link = undefined
```

Comment:
Should this be set to ''? Else we are templating undefined

Reply:
undefined is ok in printout afaik